gateway/main/settings.py

```diff
@@ -319,6 +319,7 @@
 LIMITS_CPU_PER_TASK = int(os.environ.get("LIMITS_CPU_PER_TASK", "4"))
 LIMITS_GPU_PER_TASK = int(os.environ.get("LIMITS_GPU_PER_TASK", "1"))
 LIMITS_MEMORY_PER_TASK = int(os.environ.get("LIMITS_MEMORY_PER_TASK", "8"))
+MAINTENANCE = os.environ.get("LIMITS_MEMORY_PER_TASK", "false") == "true"
```

Comment:
I think the environment variable that we want to use here is "MAINTENANCE", probably a typo.